cephfs/file_ops.go

```diff
@@ -32,3 +32,38 @@ func (mount *MountInfo) Mknod(path string, mode uint16, dev uint16) error {
 	ret := C.ceph_mknod(mount.mount, cPath, C.mode_t(mode), C.dev_t(dev))
 	return getError(ret)
 }
+
+// Utime struct is the equivalent of C.struct_utimbuf
+type Utime struct {
+	AcTime  C.time_t
+	ModTime C.time_t
+}
+
+// timespecToTime is a wrapper function that takes Timespec as its param and
+// return a struct of type *Utime.
+func timespecToTime(ts Timespec) *Utime {
```

Comment:
If this is only used in the tests, it should be moved into the _test.go file. However, see my previous comment about reusing it.